packages/@react-spectrum/contextualhelp/src/ContextualHelp.tsx

```diff
@@ -22,7 +22,10 @@ import {SlotProvider} from '@react-spectrum/utils';
 import {SpectrumContextualHelpProps} from '@react-types/contextualhelp';
 import {useMessageFormatter} from '@react-aria/i18n';
 
-function ContextualHelp(props: SpectrumContextualHelpProps) {
+/**
+ * Contextual help shows a user extra information about the state of an adjacent component, or a total view.
+ */
+export function ContextualHelp(props: SpectrumContextualHelpProps) {
```

Comment:
An oddity I noticed, we don't expose a ref through forward ref for this component.
A component that is fairly similar is ActionMenu where we do https://github.com/adobe/react-spectrum/blob/main/packages/@react-spectrum/menu/src/ActionMenu.tsx#L26

We'll need to add it in a different non-docs PR, but we should probably keep the export and comment like you originally had so that it's less code moving around

Reply:
Included this in https://github.com/adobe/react-spectrum/pull/2836, along with adding defaults to type descriptions so we can get that in before the docs go out

Reply:
so we can probably revert this now to ease the merge later